ballerina/client.bal

```diff
@@ -13,3 +16,7154 @@
+    resource isolated function get v1/quotes/[string quote](map<string|string[]> headers = {}, *GetQuotesQuoteQueries queries) returns Quote|error {
+        string resourcePath = string `/v1/quotes/${getEncodedUri(quote)}`;
+        map<Encoding> queryParamEncoding = {"expand": {style: DEEPOBJECT, explode: true}};
+        resourcePath = resourcePath + check getPathForQueryParam(queries, queryParamEncoding);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # <p>When retrieving a quote, there is an includable <a href="https://stripe.com/docs/api/quotes/object#quote_object-computed-upfront-line_items"><strong>computed.upfront.line_items</strong></a> property containing the first handful of those items. There is also a URL where you can retrieve the full (paginated) list of upfront line items.</p>
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response. 
+    resource isolated function get v1/quotes/[string quote]/computed_upfront_line_items(map<string|string[]> headers = {}, *GetQuotesQuoteComputedUpfrontLineItemsQueries queries) returns QuotesResourceListLineItems|error {
+        string resourcePath = string `/v1/quotes/${getEncodedUri(quote)}/computed_upfront_line_items`;
+        map<Encoding> queryParamEncoding = {"expand": {style: DEEPOBJECT, explode: true}};
+        resourcePath = resourcePath + check getPathForQueryParam(queries, queryParamEncoding);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # <p>When retrieving a quote, there is an includable <strong>line_items</strong> property containing the first handful of those items. There is also a URL where you can retrieve the full (paginated) list of line items.</p>
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response. 
+    resource isolated function get v1/quotes/[string quote]/line_items(map<string|string[]> headers = {}, *GetQuotesQuoteLineItemsQueries queries) returns QuotesResourceListLineItems|error {
+        string resourcePath = string `/v1/quotes/${getEncodedUri(quote)}/line_items`;
+        map<Encoding> queryParamEncoding = {"expand": {style: DEEPOBJECT, explode: true}};
+        resourcePath = resourcePath + check getPathForQueryParam(queries, queryParamEncoding);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # <p>Download the PDF for a finalized quote. Explanation for special handling can be found <a href="https://docs.corp.stripe.com/quotes/overview#quote_pdf">here</a></p>
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Successful response. 
+    resource isolated function get v1/quotes/[string quote]/pdf(map<string|string[]> headers = {}, *GetQuotesQuotePdfQueries queries) returns http:Response|error {
```

Comment:
Better to check why we are returning an HTTP response in this case and, santize if needed.

Reply:
This API allows downloading a quote as a PDF and hence we have to return an HTTP response as the payload will be a binary object.

Reply:
Ack.